assets/ajax-cart.js.liquid

```diff
@@ -39,20 +39,26 @@ ShopifyAPI.onCartUpdate = function(cart) {
 };
 
 ShopifyAPI.updateCartNote = function(note, callback) {
-  var params = {
+  var $body = $('body'),
+  params = {
     type: 'POST',
     url: '/cart/update.js',
     data: 'note=' + attributeToString(note),
     dataType: 'json',
+    beforeSend: function() {
+      $body.trigger('updateCartNote.ajaxCart', note);
```

Comment:
I changed all to before / after / error / complete events. The before trigger I didn't use but thought to add it as I was already there. :)